app/models/customer_resource.rb

```diff
@@ -22,7 +22,8 @@ class CustomerResource < RmApiResource
       request.get_params[:searchfield] = 'titlename'
       request.get_params[:orderby] = 'titlename'
       request.get_params[:count] ||= 25
-      request.get_params[:offset] ||= 1
+      request.get_params[:offset] = (request.get_params[:page] ? request.get_params[:page] : 1)
```

Comment:
Can this be an `||` operator? `request.get_params[:page] || 1`

Same goes for `package` and `title` too